mmv1/products/compute/BackendService.yaml

```diff
@@ -646,6 +652,19 @@ properties:
         description: |
           Serve existing content from the cache (if available) when revalidating content with the origin, or when an error is encountered when refreshing the cache.
         default_from_api: true
+      - !ruby/object:Api::Type::Array
+        name: 'bypassCacheOnRequestHeaders'
+        max_size: 5
```

Comment:
It is not preferred to add the validation in Terraform, in case the limit changes in the service side and it does not match with the service's requirements.